src/main/java/quotes/App.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package quotes;
+
+import com.google.gson.Gson;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class App {
+
+    // Get an array of Quote objects from the JSON file that contains each quotes
+    public static Quote[] getQuotes(String filePathToJSON) throws IOException {
+        Gson gson = new Gson();
+        String quoteFile = new String(Files.readAllBytes(Paths.get(filePathToJSON)));
+        Quote[] quotes = gson.fromJson(quoteFile, Quote[].class);
+        return quotes;
+    }
+
+    // get a random quote from the array of quotes
+    public static Quote getRandomQuote(Quote[] quotes) {
+        int randomIndex = (int)(Math.random() * quotes.length);
+        return quotes[randomIndex];
+    }
+
+    // search the array of quotes for a quote that has the author in it and return the quote
+    // Return a no such field exception if the author does not exist in the array of quotes
+    public static Quote getQuoteByAuthor(Quote[] quotes, String author) throws NoSuchFieldException {
```

Comment:
So it might be more "graceful" to catch the NoSuchFieldException since you are in the App and not a Library... and then handle that situation somehow. But I like that you are thinking of these details.